pinot-spi/src/main/java/org/apache/pinot/spi/utils/SchemaFieldExtractorUtils.java

```diff
@@ -19,17 +19,15 @@
 package org.apache.pinot.spi.utils;
 
 import com.google.common.annotations.VisibleForTesting;
-import com.google.common.base.Preconditions;
-import java.util.ArrayList;
 import java.util.HashSet;
 import java.util.List;
 import java.util.Set;
+import org.apache.pinot.spi.data.function.evaluators.ExpressionEvaluator;
+import org.apache.pinot.spi.data.function.evaluators.ExpressionEvaluatorFactory;
```

Comment:
there are two methods here `extract` and `extractSource`. I believe `extractSource` is for testing purpose only. Can you move it to the test, or in some way, get rid of it (e.g. maybe by contriving the data in test schema). There are no comments on either of these methods, and it can be confusing. 
I understand it came from a previous commit, but please fix it. 
Thanks

Reply:
Removed the test method, kept only one. Also s/extract/extractSourceFields